docs/docs/recipes/working-with-starters.md

```diff
@@ -1,38 +1,38 @@
 ---
-title: "Recipes: Working with Starters"
-tableOfContentsDepth: 1
+শিরোনাম: "রেসিপি: স্টার্টারের সাথে কাজ করা"
+টেবিলঅফ বিষয়বস্তুর গভীরতা: 1
 ---
 
-[Starters](/docs/starters/) are boilerplate Gatsby sites maintained officially, or by the community.
+[স্টার্টার](/docs/starters/) বয়লারপ্লেট গ্যাটসবি সাইটগুলি সরকারীভাবে বা সম্প্রদায় দ্বারা রক্ষণাবেক্ষণ করা হয়.
```

Comment:
I don't think this is the correct/appropriate translation for the line.
We should use something more suitable for the words `officially` and `community`.